### DISCUSSION TITLE
第1节课笔记(20班)

### DISCUSSION THREAD

**Discussion by Farewell-CK** [InternLM/Tutorial#200] [笔记]
宝子们，第一节课，20班的笔记交到这里哦

**Comment by TLzh** [comment]
笔记链接：https://www.jianshu.com/p/75c44d3466d5?v=1704785997366

**Reply by TLzh** [reply]
![图片](https://github.com/InternLM/tutorial/assets/22994534/cf42125e-3b7d-42db-a475-7985bad63072)
![图片](https://github.com/InternLM/tutorial/assets/22994534/5fc57a58-e6ed-43f6-a59d-414f3fc381c5)

啊这，笔记被锁定了

**Reply by Farewell-CK** [reply]
解决了嘛

**Reply by TLzh** [reply]
新的链接
http://t.csdnimg.cn/KOhfi

**Comment by Walshyu** [comment]
https://github.com/Liltomato1998/hello-world/blob/main/LLM/%E7%AC%AC%E4%B8%80%E8%8A%82%E8%AF%BE%E7%AC%94%E8%AE%B0.pdf

**Comment by deyouzuishuai** [comment]
笔记链接：https://zhuanlan.zhihu.com/p/676912650

**Comment by LiyanJin** [comment]
https://blog.csdn.net/weixin_53443275/article/details/135468011

**Comment by liu673** [comment]
笔记链接：https://github.com/liu673/InternLM_OpenNotebook/blob/master/lesson_1/lesson_1_note.md

**Comment by 20300240011** [comment]
笔记链接：https://zhuanlan.zhihu.com/p/677275746

**Comment by TtKkD** [comment]
笔记连接：http://t.csdnimg.cn/8Z2A6

**Comment by qiaoxiaopang** [comment]
笔记链接：https://blog.csdn.net/weixin_74322438/article/details/135536089

**Comment by CoconutCC** [comment]
笔记链接：https://blog.csdn.net/weixin_54741574/article/details/135537653

**Comment by forestemperor** [comment]
笔记链接：https://github.com/forestemperor/InternLM_studynote/blob/main/stydynote/lesson1.md

**Comment by zhangyiyuan1112** [comment]
笔记链接：https://iqxyis6ody.feishu.cn/docx/ZDEmdgfkvoXzHBx8q8ccyqH3nmg?from=from_copylink

**Comment by Nobody-ML** [comment]
https://zhuanlan.zhihu.com/p/677353550

**Comment by pooh0315** [comment]
笔记链接：https://blog.csdn.net/m0_74206611/article/details/135541016

**Comment by yalunren** [comment]
https://docs.qq.com/doc/DZHNnUXVtY25NdkFo?u=ccf8085364c54d099fd089ceaab4fa20
